packages/@react-spectrum/tag/docs/TagGroup.mdx

```diff
@@ -112,14 +114,39 @@ function Example() {
 
   return (
     <Flex width={500}>
-      <TagGroup items={items} allowsRemoving onRemove={removeItem} aria-label="Removable TagGroup example">
+      <TagGroup
+        items={items}
```

Comment:
Let's reduce the number of items in this example so that they don't take up so much space in the code (especially on mobile)